### PR TITLE
[ONNX] Remove aten parameter (#61652)

### PR DESCRIPTION
**Pull request by BowenBao**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #63249 Enhance shape (#60335)
* #62765 [ONNX] Suppport torch.dot and torch.nn.utils.spectral_norm (#62596)
* #62764 [ONNX] Update repeat_interleave for dynamic repeats (#59979)
* #62763 [ONNX] Fix an issue that optimizations might adjust graph inputs unexpectedly. (#61280)
* #62762 [ONNX] Fix controlflow shape inference with contrib op (#60707)
* #62760 [ONNX] Fix for batchnorm training op mode (#52758)
* **#62759 [ONNX] Remove aten parameter (#61652)**
* #62758 [ONNX] Add support for opset14 in PT-ONNX exporter (#59486)
* #62757 [ONNX] Support lstm_cell symbolic (#61476)

* remove aten argument in export()

* add export_to_pretty_string default value OperatorExportTypes.ONNX

* add DPYTORCH_ONNX_CAFFE2_BUNDLE description

Co-authored-by: Gary Miguel <garymiguel@microsoft.com>

Differential Revision: [D30349062](https://our.internmc.facebook.com/intern/diff/D30349062)

### CODE REVIEW

**Comment by SplitInfinity** [comment]
@splitinfinity has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30349062).

**Comment by msaroufim** [comment]
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30349062).
